libflux/flux-core/src/ast/mod.rs

```diff
@@ -1483,7 +1485,7 @@ pub struct Property {
 }
 
 /// Identifier represents a name that identifies a unique Node
-#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+#[derive(Debug, Default, PartialEq, Clone, Serialize, Deserialize)]
```

Comment:
I don't think I knew you could derive `Default`. What does it return when you call `Indentifier::default()`?